README.md

```diff
@@ -1,24 +1,75 @@
 # Go-TPM tools
 
-This repository contains various libraries and a command line tool designed for
-use with [Go-TPM](https://github.com/google/go-tpm):
-  - [simulator](https://godoc.org/github.com/google/go-tpm-tools/simulator):
-    Go bindings to the Microsoft's
-    [TPM2 simulator](https://github.com/Microsoft/ms-tpm-20-ref/).
-  - [tpm2tools](https://godoc.org/github.com/google/go-tpm-tools/tpm2tools):
-    a Go library providing useful abstractions and utility functions for using a
-    TPM2. The goal of this library is to handle complex TPM functionality
-    (sessions, authorization, activating credentials, etc...), providing users
-    with a simplified API.
-  - `gotpm`: a command line tool for using the TPM from the command line. Run
-    `gotpm --help` and `gotpm [command] --help` for more documentation.
+This repository contains various libraries designed for use with [Go-TPM](https://github.com/google/go-tpm):
+  - [`client`](https://pkg.go.dev/github.com/google/go-tpm-tools@v0.3.0-alpha/client):
+    A client-side Go library providing simplified TPM abstractions and utility functions. This includes:
```

Comment:
Drop the "client-side". Something like: "A Go library that simplifies interactions with a TPM2 by providing utility functions and abstractions, including:"